ser/data.py

```diff
@@ -0,0 +1,24 @@
+from pathlib import Path
+from torch.utils.data import DataLoader
+from ser.transforms import get_transforms
+from torchvision import datasets
+
+
+PROJECT_ROOT = Path(__file__).parent.parent
+DATA_DIR = PROJECT_ROOT / "data"
+
+def get_data_loaders(batch_size = 2): 
```

Comment:
that's a very small default batch size :) Obviously this is overwritten by the cli default but a default here in the function definition might confusing a user (if it's non-standard)